src/gen_rpc_client.erl

```diff
@@ -148,6 +165,21 @@ safe_cast(Node, M, F, A, SendTO) when is_atom(Node), is_atom(M), is_atom(F), is_
             gen_server:call(Pid, {{cast,M,F,A},SendTO}, infinity)
     end.
 
+%% Safe evaluate Module:Function:Arguments on implicit connected nodes. Custom sender timeout.
+safe_eval_everywhere(Nodes, M, F) ->
+    safe_eval_everywhere(Nodes, M, F, [], 'undefined').
+
+%% Safe evaluate Module:Function:Arguments on implicit connected nodes. Custom sender timeout.
+safe_eval_everywhere(Nodes, M, F, A) ->
+    safe_eval_everywhere(Nodes, M, F, A, 'undefined').
+
+
+%% Safe evaluate Module:Function:Arguments on custom list of nodes.
+safe_eval_everywhere(Nodes, M, F, A, SendTO) ->
+    ok = lager:debug("function=safe_eval_everywhere_mfa_to event=eval_on_nodes nodes=\"~p\"", [Nodes]),
+    [{Node, safe_cast(Node, M, F, A, SendTO)} || Node <- Nodes].
```

Comment:
Why return a tuple here? The function is supposed to return just the results in proper order. Also, per the usual "multicall" paradigm, this should return {true, BadNodes} where BadNodes are the nodes with badtcp/badrpc results